docs/PyOp.md

```diff
@@ -0,0 +1,71 @@
+# Python Operator 
+To facilitate Python coders on model developing, onnxruntime provides a way to invoke operators implemented in Python.
+
+## Usage
+Step 1, build onnxruntime with“--config Release --enable_language_interop_ops --build_shared_lib”and override existing onnxruntime binary with the latest, then copy onnxruntime_pywrapper.dll or libonnxruntime_pywrapper.so or libonnxruntime_pywrapper.dylib to the path where onnxruntime binary is placed.
```

Comment:
Can we reuse the enable_python_bindings to enable this capability? The current config in the command line looks pretty irrelevant.

Reply:
The purpose of adding new flags is to make sure user know exactly what he wants.
Python bindings does not necessarily mean a user is expecting pyop.
Conversely, pyop does not require python binding at all.

Reply:
I think it is better this way, otherwise what would happen with the C or C# API?

Reply:
Yes, it would happend in C# API - if someone is using C# API and he want to also go with PyOP.